IBMWatsonAssistantV1.podspec

```diff
@@ -14,10 +14,12 @@ natural-language input and uses machine learning to respond to customers in a wa
 
   s.module_name           = 'Assistant'
   s.ios.deployment_target = '8.0'
-  s.source                = { :git => 'https://github.com/watson-developer-cloud/swift-sdk.git', :tag => "v#{s.version.to_s}" }
+  s.source                = { :git => 'https://github.com/watson-developer-cloud/swift-sdk.git', :tag => s.version.to_s }
   
-  s.source_files          = 'Source/AssistantV1/**/*.swift'
+  s.source_files          = 'Source/AssistantV1/**/*.swift',
+                            'Source/SupportingFiles/Shared.swift'
+  s.exclude_files         = 'Source/AssistantV1/Shared.swift'
```

Comment:
This is necessary because there are symlinks for `Shared.swift` in each service's source folder. Unlike Carthage, Cocoapods isn't able to get the contents of `Shared.swift` from a symlink. If I include the one from the SupportingFiles folder, then Cocoapods complains that there are 2 files with the same name (weird, I know). So this is the only way it works.